testsuite/forge/src/backend/k8s/node.rs

```diff
@@ -132,17 +130,10 @@ impl Node for K8sNode {
     }
 
     fn health_check(&mut self) -> Result<(), HealthCheckError> {
-        let client = BlockingClient::new(self.json_rpc_endpoint());
-        let results = match client.batch(Vec::new()) {
-            Ok(x) => x,
-            Err(x) => return Err(HealthCheckError::RpcFailure(format_err!(x))),
-        };
-        if results.iter().all(Result::is_ok) {
-            return Ok(());
-        }
-        Err(HealthCheckError::RpcFailure(format_err!(
-            "K8s node health_check failed"
-        )))
+        reqwest::blocking::get(self.rest_api_endpoint()).map_err(|e| {
```

Comment:
probably better to use the actual rest client here, wrapped in a block_on call till we change these apis to be async

Reply:
I'll need keep this for now this PR, and maybe workout a solution later.
Because the call maybe inside a runtime already, which leads to error "thread 'main' panicked at 'Cannot start a runtime from within a runtime", and I think there are also calls that is not inside a runtime, so simply assume it's inside a runtime call does not work as well.